### PR TITLE
Docs/update documentation

### PR DESCRIPTION
**Pull request by JFCowboy**
## Description

[//]: # (Please inlcude a summary of the change and which issues is fixed. Please also include relavant motivation and context. List any dependencies that are required fo change.)

Update of spec and asyncapi

## Trello link:

https://trello.com/c/VBfhzjKx/77-actualiza-la-definicion-de-interfaces

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Checklist
[//]: # (Go over all the following points, and put an `x` in all the boxes that apply.)
[//]: # (If you're unsure about any of these, don't hesitate to ask.)

- [x] Changes were successfully tested.
- [ ] Dependencies. All dependencies are deployed.

## How Has This Been Tested?

[//]: # (Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration)

### CODE REVIEW

File: docs/preguntonicAsyncapi.yml [hunk 1]
```diff
@@ -1,21 +1,23 @@
 asyncapi: 3.0.0
 info:
   title: Preguntonic API
-  version: 1.0.0
+  version: 1.2.0
   description: |-
-    Preguntonic is a simple API that allows you to ask questions and get answers in real-time.
-    With Preguntonic you can:
+    Preguntonic s una API en tiempo real diseñada para sesiones interactivas de preguntas y respuestas. Ofrece una plataforma dinámica donde puedes:
```
Comment (peenyaa7) [inline docs/preguntonicAsyncapi.yml]:
```suggestion
    Preguntonic es una API en tiempo real diseñada para sesiones interactivas de preguntas y respuestas. Ofrece una plataforma dinámica donde puedes:
```

**Review by peenyaa7 — APPROVED** [review]

**Comment by peenyaa7** [comment]
@JFCowboy check last commit 👌

File: README.md [hunk 2]
```diff
@@ -9,27 +10,66 @@ PreguntonicBackend es el núcleo inteligente y robusto que impulsa la experienci
...
-    docker compose up
-    ```
-4. open the browser and go to http://localhost:8080
+### With Docker 🐳
+
+Create `.jar` file
+
+```bash
+mvn clean package
+```
```
Comment (JFCowboy) [inline README.md]: With new docker this step is not necessary but this is still nedded:

```
2. build the docker image
    ```bash
    docker compose build preguntonic-backend
    ```
```

**Review by Alejandro25AR — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -9,27 +10,66 @@ PreguntonicBackend es el núcleo inteligente y robusto que impulsa la experienci
...
+
+```bash
+  npm install
+```
+
+Start the server
+
+```bash
+  npm run start
+```
```
Comment (JFCowboy) [inline README.md]: is this really needed?

**Pull request merged**